src/atoms/Button/buttons.module.scss

```diff
@@ -57,191 +11,61 @@ $brand-white: color('neutral-8');
 }
 
 .button {
-  @include typography-6;
-  @include typography-semibold;
-  @include hover($button-hover-background-color, $button-border-color, $button-hover-color);
+  @include typographyA(8);
 
   display: block;
   cursor: pointer;
   text-align: center;
   text-decoration: none;
-  color: $button-color;
+  color: $brand-white;
   margin: 0;
-  padding: ru(.5);
-  background-color: $button-background-color;
-  border: 1px solid $button-border-color;
-  border-radius: $button-border-radius;
+  padding: rem-calc(19);
+  background-color: $action-color;
+  border: 2px solid $action-color;
   transition: background-color 300ms ease-out,
   border-color 300ms ease-out,
   top 150ms ease-out,
   color 150ms ease-out,
   box-shadow 150ms ease-out;
   width: 100%;
   font-size: ru(.5);
+  text-transform: uppercase;
 
   .icon {
     path {
       transition: fill 300ms ease-out;
-      fill: $button-color;
+      fill: $action-color;
```

Comment:
is the `Icon` only supposed to be used on `outline` Buttons? The first example is using the `Icon` on the solid orange button and you can't see it. Can you provide some clarity here/update the Readme?

Reply:
Fair... I moved through this quickly. We no longer have buttons with Icons, so I just left it.

Reply:
Updating to make the icon white.